src/main/groovy/org/shipkit/internal/notes/improvements/GitHubTicketFetcher.java

```diff
@@ -151,7 +153,7 @@ public GitHubIssuesBuilder labels(String labels) {
             GitHubIssues browse() {
                 // see API doc: https://developer.github.com/v3/issues/
                 String nextPageUrl = String.format("%s%s%s%s%s%s%s",
-                        "https://api.github.com/repos/" + repository + "/issues",
+                        apiUrl + "/repos/" + repository + "/issues",
```

Comment:
I don't like the usage of String.format here, it doesn't give us anything when it only contains variables. Would be better to have a simple String concatenation and any more complex logic could be extracted into private methods, like baseUrl(), state(), filter(), direction().